tgui/packages/tgui/interfaces/SmartFridge.tsx

```diff
@@ -0,0 +1,175 @@
+  context
+) => {
+  const [tabIndex, setTabIndex] = useLocalState(
+    context,
+    `contentsTab_${props.isLocal}`,
+    'all'
+  );
+  const allItems = props.items;
+
+  if (allItems.length === 0) {
+    return (
+      <Section title={props.title}>
+        <span>No items present</span>
+      </Section>
+    );
+  }
+
+  const categories = new Map<string, StorageItem[]>();
+  categories.set('all', allItems);
+  allItems.forEach((x) => {
+    const categoryArray = categories.get(x.category) ?? new Array();
+    categoryArray.push(x);
+    categories.set(x.category, categoryArray);
+  });
+
+  const categoryIterable = Array.from(categories.entries());
+  return (
+    <Section title={props.title}>
+      <Tabs fill fluid>
+        {categoryIterable
+          .sort((a, b) => a[0].localeCompare(b[0]))
+          .map((value) => {
+            const key = value[0];
+            const items = value[1];
+            return (
+              <Tabs.Tab
+                key={key}
+                selected={tabIndex === key}
+                onClick={() => setTabIndex(key)}>
+                {key} ({items.length})
+              </Tabs.Tab>
+            );
+          })}
+      </Tabs>
+
+      {categoryIterable.map((x) => {
+        if (tabIndex !== x[0]) {
+          return undefined;
+        }
+        return (
+          <ContentsTable key={x[0]} isLocal={props.isLocal} items={x[1]} />
+        );
+      })}
+    </Section>
+  );
+};
+
+const ContentItem = (
+  props: { isLocal: boolean; item: StorageItem },
+  context
+) => {
+  const { data, act } = useBackend<SmartFridgeData>(context);
+  const { item } = props;
+  const itemref = { 'index': item.index, 'amount': 1, isLocal: props.isLocal };
+  return (
+    <>
+      <TableCell className="ItemIconCell">
+        <span
+          className={classes(['ItemIcon', `vending32x32`, `${item.image}`])}
+        />
+      </TableCell>
+      <TableCell className="ItemIconCell">{item.quantity}</TableCell>
+      <TableCell>
+        <Button
+          className="VendButton"
+          preserveWhitespace
+          textAlign="center"
+          icon="circle-down"
+          onClick={() => act('vend', itemref)}>
+          {item.display_name}
+        </Button>
+      </TableCell>
+      {data.networked === 1 && (
+        <TableCell>
+          <Button
+            icon={props.isLocal ? 'upload' : 'download'}
+            onClick={() => act('transfer', itemref)}
+          />
+        </TableCell>
+      )}
+      <TableCell>
+        <Icon name="circle-info" />
+      </TableCell>
+    </>
+  );
+};
+
+export const SmartFridge = (_, context) => {
+  const { data } = useBackend<SmartFridgeData>(context);
+  return (
+    <Window theme="weyland" width={400} height={600}>
+      <Window.Content className="SmartFridge" scrollable>
+        <Stack vertical>
+          {data.secure && (
+            <Stack.Item>
+              <NoticeBox>Smart fridge is secure</NoticeBox>
```

Comment:
```suggestion
              <NoticeBox>Smart Fridge is on Secured Mode</NoticeBox>
```
or something to that effect

Reply:
sounds good, went with Smart Fridge is in secure mode